appstudio-controller/controllers/appstudio.redhat.com/application_controller.go

```diff
@@ -274,6 +274,12 @@ func getGitOpsRepoData(asApplication applicationv1alpha1.Application) (string, s
 		return "", "", "", fmt.Errorf("gitops repo context is empty: %v", err)
 	}
 
+	// Argo CD expects a "non-absolute" path here. Argo CD interprets "/" as absolute, so change it to "."
+	// to indicate the root.
+	if context == "/" {
+		context = "."
```

Comment:
is there any way to test whart ArgoCD expects? e.g. what happens if the context equals `./` or `./..`?